lambda/voting/artifacts/contracts/Voting.sol/Voting.json

```diff
@@ -7,13 +7,232 @@
       "inputs": [
         {
           "internalType": "address",
-          "name": "owner",
+          "name": "lambdaOwner",
```

Comment:
actually remove this file and update the deploy command to run hardhat compile first

Reply:
done

Reply:
done